openapi/swagger.yml

```diff
@@ -797,6 +813,22 @@ paths:
         name: program
         required: true
         type: string
+      - description: 'category of node to filter on children. Example: clinical'
+        in: query
+        name: category
+        type: string
+      - description: whether to recursively find children or not; default is False
+        in: query
+        name: with_children
+        type: string
+      - description: one or a list of gdc ids seperated by commas.
```

Comment:
couple nitpicks:
- `gdc ids` could just be `node IDs`
- the ` ``monospace`` ` syntax is for Sphinx specifically; swagger uses only a single backtick pair: ` `json` `